script/licensed

```diff
@@ -24,10 +22,10 @@ module Licensed
 
       def dependencies
         Dir.glob(@glob).map do |directory|
-          puts "caching #{directory}"
```

Comment:
This is duplicating info that is already printed by Licensed. Licensed will print:

```
Caching licenses for linguist:
  grammar dependencies:
    Using sublime-rexx (2e304a34d3493e7453a430735191d91f356faa5a)
    Using Modelica (c841b17b0f441451b43a0b0e62bc6201a3456b83)
    Using atom-language-purescript (cc2cb471968eb331af7259aac219feb3017d709e)
[...]
    Caching Sublime-Coq (94e43bdc194535cf5bae9306624039af9f38c014)
[...]
```

`Using` signifies nothing has changed, `Caching` signifies a change.